docs/concepts/explore-available-balancer-pools/reclamm-pool/reclamm-pool-math.md

```diff
@@ -0,0 +1,337 @@
+
+$balanceRatio = \frac{R_b'}{R_a'}$
+
+### Initialization
+
+Finally, when initializing, we need to scale $V_a'$ and $V_b'$ to the initial balances of the pool. The scale is $\frac{R_a}{R_a'}$ (which is the same rate as $\frac{R_{a_{max}}}{R_{a_{max}}'}$). So
+
+$V_a = \frac{R_aV_a'}{R_a'}, V_b = \frac{R_aV_b'}{R_a'}$
+
+## Price Ratio ($Q_0^2$)
+
+The Price Ratio is the ratio between the high and low price bounds of the range. The current price of token A is defined as $P_A = \frac{R_b + V_b}{R_a + V_a}$.
+
+The highest price of A ($P_{max_A}$) is reached when the real balance of A is 0. So, the price is defined as:
+
+![ReClamm price equation](/images/reclamm-price-equation.png)
+
+The Price Ratio is calculated as $Q_0^2 = \frac{P_{max_A}}{P_{min_A}}$.
+
+For example, let's say the pool has virtual balances of 1000 of each token. Also, let's assume that when the real balance of token A is 0, token B is 1000 ($R_{max_{B}}$), and vice versa.
+
+In the example, $P_{max_{A}} = \frac{1000 + 1000}{1000} = 2$ and $P_{min_{A}} = \frac{1000}{1000 + 1000} = 0.5$, so $Q_0^2 = \frac{2}{0.5} = 4$.
+
+### Price Ratio Update
+
+The Price Ratio can be updated by a pool manager. This update is gradual, analogous to updating the amplification factor of a Stable Pool. The pool manager defines a start and end time and a target $Q_{0_{target}}$, and then $Q_{0_{new}}$ is calculated as an interpolation between the initial and final values, $Q_{0_{initial}}$ and $Q_{0_{target}}$.
+
+$\sqrt{Q_{0_{new}}} = \sqrt{Q_{0_{initial}}}(\frac{\sqrt{Q_{0_{target}}}}{\sqrt{Q_{0_{initial}}}})^{\frac{blockTimestamp - startQ0Time}{endQ0Time - startQ0Time}}$
+
+## Pool Centeredness
+
+As noted above, the main idea of this pool is to manage the price interval such that the market price stays within it as much as possible. The pool won't use an external oracle, so we need a mechanism to determine whether the market price is within the current price interval: `Pool Centeredness`.
+
+`Pool Centeredness` is a number (a percentage) that describes how far the pool balances are from the center of the current interval. The number goes from 0 to 1, where 0 means that the balances are at the edge and 1 means that the balances are exactly in the middle of the current price range, which is the case when the pool is initialized. The number is calculated as follows:
+
+1. First, if $R_a$ or $R_b$ are 0, we know that pool centeredness is 0.
+2. If not, we calculate $\frac{R_a}{R_b}$ and compare with $\frac{V_a}{V_b}$. Since the virtual balances were calculated based on the real balances, these rates are the same near the center of the interval.
+    1. If $\frac{R_a}{R_b} < \frac{V_a}{V_b}$, $centeredness = \frac{\frac{R_a}{R_b}}{\frac{V_a}{V_b}}=\frac{R_a V_b}{R_b V_a}$
+    
+    2. Otherwise, $centeredness = \frac{\frac{R_b}{R_a}}{\frac{V_b}{V_a}}=\frac{R_b V_a}{R_a V_b}$
```

Comment:
In the contract, this logic is slightly different, we compare `Ra Vb` with `Rb Va` since these multiplication operations have more precision than the division. But, honestly, I think explaining like this is more intuitive, so I'd leave as is.

Anyway, maybe it's worth it putting a note than the contract implementation uses multiplications to be more precise? (Or, if the idea is to show only the logic, I'd leave as is)